config/keyprovider-config/config.go

```diff
@@ -0,0 +1,68 @@
+/*
+   Copyright The ocicrypt Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package keyprovider_config
+
+import (
+	"encoding/json"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"os"
+)
+
+type OcicryptConfig struct {
+	KeyProviderConfig map[string]interface{} `json:"key-providers"`
```

Comment:
How about making this a struct to include

```
type Command struct{...}
type Grpc struct{...}
```

and then it can be referenced later in the KeyWrapper

Reply:
done